src/common/Common.ts

```diff
@@ -18,6 +18,55 @@ import { CustomOptions, OptionFlag } from '@oclif/core/interfaces';
 Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
 const messages = Messages.loadMessages('@salesforce/lwc-dev-mobile-core', 'common');
 
+export class CaseInsensitiveStringMap {
```

Comment:
This is essentially a `Map` where you can index to it using `key` that is case insensitive... this is used when parsing adb output during device discovery phase and makes things more robust.